portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/*
+ * Runs two functions on load
```

Comment:
Make this one line if it fits easily on one line.

Reply:
done